### DISCUSSION TITLE
Cant send welcoming email

### DISCUSSION THREAD

**Discussion by lakomyt** [requarks/wiki#4876] [Help / Questions] [answered]
This option is grayed out. I've configured mail account and it works
![Screenshot from 2022-01-11 15-18-07](https://user-images.githubusercontent.com/37346583/148961907-e08970fa-9871-4edf-ba33-cb654fc8ab02.png)

**Comment by NGPixel** [comment, marked as answer]
It's greyed out because it's not available yet.

**Reply by db6edr** [reply]
Sorry for necrobumping, but: WTF is that option visible without any explanation why it is disabled?
